### DISCUSSION TITLE
Bug: roundRect doesn't have equal rounded corners

### DISCUSSION THREAD

**Discussion by markknol** [pixijs/pixijs#10961] [Q&A] [answered]
### Current Behavior

Rounded corners of a rectangle are not looking similar on all edges.
![image](https://github.com/user-attachments/assets/8b163955-88c6-4198-8876-48eb74994f0c)

### Expected Behavior

The edges of the rounded rectangle should all have similar looking rounded corners.

### Steps to Reproduce

Add a `new Graphics().roundRect(0, 0, 140, 24, 5).fill('#333').stroke('#fff')`

### Environment

- **`pixi.js` version**: 8.3.4
- **Browser & Version**: Chrome 129.0.6668.60
- **OS & Version**: Windows
- **Running Example**: <https://pixiplayground.com/#/edit/vJuHK_dnsps8_RKuYyh9H>

**Comment by bigtimebuddy** [comment, marked as answer]
This is not a bug, what you are seeing here is aliasing of the graphics which, by default, are pixelated looking. if you want to antialias there are a few ways to do that:

* [Set `antialias: true` on the Application](https://pixiplayground.com/#/edit/QT0y4jnB8oKwtWha1S_sy)
* [Use a filter and set `antialias` to `true`](https://pixiplayground.com/#/edit/NaWTcFf8Y_MF1-UPxJSWE)

**Comment by markknol** [comment]
Thanks for the reply!
Interesting, and yes with antialias it looks great. I was just suprised it looks very asymmetrical without aliassing. This makes it less usable for pixelart games.
